### PR TITLE
draft blog post contributing guidelines

### PR DESCRIPTION
**Pull request by shannonpileggi**

### CODE REVIEW

**Comment by shannonpileggi** [comment]
@yabellini i ammeded the workflow instructions to say use whatever method you are comfortable with etc, instead of writing out another set of instructions. we can always add later if it is a common ask, but i think this is sufficient for now. do you agree?

i also ammended the wording with regards to knitting vs site rendering.

**Comment by yabellini** [comment]
Looks good to me. Thanks so much!

**Comment by shannonpileggi** [comment]
thanks @yabellini! we need at least one reviewer to approve before we merge.

**Review by yabellini — APPROVED** [review]
Thanks so much for writing this guidelines!

File: content/comm/blog/index.en.md
Comment (drmowinckels) [inline content/comm/blog/index.en.md:15]:
```suggestion
- Athanasia M. Mowinckel [`@drmowinckels`](https://github.com/drmowinckels)
```

File: content/comm/blog/index.en.md
Comment (drmowinckels) [inline content/comm/blog/index.en.md:19]:
```suggestion
- Athanasia M. Mowinckel [`@drmowinckels`](https://github.com/drmowinckels)
```

File: content/comm/blog/index.en.md
Comment (drmowinckels) [inline content/comm/blog/index.en.md:131]:
```suggestion
```

File: content/comm/blog/index.en.md
Comment (drmowinckels) [inline content/comm/blog/index.en.md:151]:
```suggestion
on the site, not your `Rmd` itself.  If your post does not include any code chunks 
(i.e. no code is being run), you can write a single `.md` file instead, and not need
to knit at all.
```

**Review by drmowinckels — APPROVED** [review]
Fixes my user links a couple places, and a couple of extra suggestions.

**Comment by shannonpileggi** [comment]
@drmowinckels i think this is good to merge once checks pass

**Pull request merged**
